src/test/java/org/wikivoyage/listings/input/DumpDownloaderTest.java

```diff
@@ -0,0 +1,35 @@
+package org.wikivoyage.listings.input;
+
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import org.apache.commons.io.IOUtils;
+import org.junit.Assert;
+import org.junit.Test;
+import org.wikivoyage.listings.input.DumpDownloader;
+
+public class DumpDownloaderTest {
+
+	/**
+	 * Test if the partial dump is detected correctly
+	 */
+	@Test
+	public void isNotPartialDump() {
+		String dumpStatus="", partialDumpStatus="";
+	    try {
+	    		dumpStatus = IOUtils.toString(
+	                this.getClass().getResourceAsStream("/dumpstatus.json"), "UTF-8"
+	            );
+	    		partialDumpStatus = IOUtils.toString(
+		                this.getClass().getResourceAsStream("/partial_dumpstatus.json"), "UTF-8"
+		            );
+	    		DumpDownloader downloader = new DumpDownloader();
+	    		Assert.assertFalse(downloader.isPartialDump(dumpStatus));
+	    		Assert.assertTrue(downloader.isPartialDump(partialDumpStatus));
+		} catch (IOException e) {
```

Comment:
Generally you don't need to catch exception in unit tests, besides cases where exceptions are the topic of the test.
Just adding `throws Exception` to the prototype makes for clearer code.
An exception thrown makes the test fail.